source/connection-string/tests/valid-warnings.yml

```diff
@@ -86,3 +86,6 @@ tests:
         auth: ~
         options:
             authMechanism: MONGODB-OIDC
+            authmechanismProperties:
```

Comment:
This seems incorrect to me.  There should be no returned authMechanismProperties, since it fails to parse correctly.  Also, this TOKEN_RESOURCE doesn't match the one the uri.    Tested with https://spruce.mongodb.com/task/mongo_python_driver_test_rhel8_python3.10_test_4.4_standalone_noauth_nossl_sync_async_patch_eaae22c63ba6673460d18c7782b65716f9e18ea4_67bf16a8a0f2a800076ea1e0_25_02_26_13_27_22/logs?execution=0&sortBy=STATUS&sortDir=ASC

Reply:
ah this was a bad copy/paste, removed. (we treat warnings in Rust as failures so the test runner wasn't doing any comparison here)